src/Faker/Provider/id_ID/Person.php

```diff
@@ -244,13 +285,68 @@ public static function lastNameFemale()
     }
 
     /**
-     * For academic title
+     * For academic title.
      *
-     * @access public
      * @return string suffix
      */
     public static function suffix()
     {
         return static::randomElement(static::$suffix);
     }
+
+    /**
+     * Return vehicle license (police) number.
+     *
+     * @return string vehicle license number
+     */
+    public static function vehicleLicenseNumber()
+    {
+        return static::personalLicenseNumber();
+    }
+
+    /**
+     * Return vehicle license (police) number.
+     *
+     * @return string vehicle license number
+     */
+    public static function personalLicenseNumber()
+    {
+        $license = static::randomElement(static::$vehicleAreaCode);
```

Comment:
better: use `$this->generator->parse()` with a format string. See other providers as a usage example.